index.js

```diff
@@ -687,4 +687,18 @@ export const mentorshipTeam = [
       },
     ],
   },
+  {
+    name: 'Vien Nguyen',
+    title: 'Student @ University of South Florida',
+    hobbies: 'Fencing, table tennis, travelling',
+    avatar:
+      'https://media.licdn.com/dms/image/D4D03AQFtTFeQ82PChA/profile-displayphoto-shrink_800_800/0/1695135628848?e=1717027200&v=beta&t=r4i7BAQaMqpvjjNjCTCDoNJQ7Q5IcCmrRnLDcWaR96s',
```

Comment:
update url to `https://res.cloudinary.com/khoa165/image/upload/v1711819446/viettech/vien.jpg`